addons/survey/models/survey_question.py

```diff
@@ -650,6 +650,24 @@ def _get_stats_summary_data_scored(self, user_input_lines):
             'right_inputs_count': len(user_input_lines.filtered(lambda line: line.answer_is_correct).mapped('user_input_id'))
         }
 
+    # ------------------------------------------------------------
+    # UTILS METHODS
+    # ------------------------------------------------------------
+
+    def get_correct_answer(self):
+        ''' Get the question correct answer(s) (only for scorable question types). '''
+        if self.question_type in ['numerical_box', 'date', 'datetime']:
+            return self[f'answer_{self.question_type}']
+        elif self.question_type in ['simple_choice', 'multiple_choice']:
+            return self.suggested_answer_ids.filtered(lambda sug: sug.is_correct).ids
+
+    @api.model
+    def get_correct_answers(self, question_ids):
```

Comment:
Note that this highlights there is an issue with the answer validation: either people cannot access questions and it will crash, either they can and we have a leaking issue ^^ (see previous comment)